scripts/cnv_wdl/germline/gCNV_panel_creation_workflow.wdl

```diff
@@ -0,0 +1,191 @@
+# Workflow for creating a panel of normals for germline CNV pipeline
+# Notes: 
+#
+# -Basic sex genotype tab separated table for homo sapiens must be formatted as follows (See SexGenotypeTableReader.java class for full description):
```

Comment:
Same comments as those on `gCNV_cohort_calling_workflow.wdl`. Also, please review the formatting (line breaks before and after example tables, wording, etc.) all other.